src/Application/Regulation/Command/SaveMeasureCommandHandler.php

```diff
@@ -66,8 +67,12 @@ public function __invoke(SaveMeasureCommand $command): Measure
                     $locationsNewStillPresentUuids[] = $locationNewCommand->locationNew->getUuid();
                 }
 
-                $locationNewCommand->measure = $command->measure;
-                $this->commandBus->handle($locationNewCommand);
+                try {
+                    $locationNewCommand->measure = $command->measure;
+                    $this->commandBus->handle($locationNewCommand);
+                } catch (GeocodingFailureException $e) {
```

Comment:
Ouais j'aime bien ça, en tout cas en Python c'est un pattern encouragé, au nom de "explicit is better than implicit". Ça permet de voir dans une fonction donnée tous les endroits où elle pourrait throw, même si c'est parce qu'un appel interne a throw qqc.